bricksrc/collections.py

```diff
@@ -83,3 +108,30 @@
     },
     "Shading_System": {"tags": [TAG.Shade, TAG.System]},
 }
+
+loop_subclasses = {
+    "Air_Loop": {"tags": [TAG.Air, TAG.Loop]},
+    "Hot_Water_Loop": {"tags": [TAG.Hot, TAG.Water, TAG.Loop]},
+    "Chilled_Water_Loop": {"tags": [TAG.Chilled, TAG.Water, TAG.Loop]},
+}
+
+collection_classes = {
+    "Portfolio": {
+        "tags": [TAG.Collection, TAG.Portfolio],
+        "constraints": {BRICK.hasPart: [BRICK.Site]},
+    },
+    "System": {
+        "tags": [TAG.Collection, TAG.System],
+        "subclasses": system_subclasses,
+        "constraints": {BRICK.hasPart: [BRICK.Equipment, BRICK.Point, BRICK.Loop]},
+    },
+    "Loop": {
+        "tags": [TAG.Collection, TAG.Loop],
+        "subclasses": loop_subclasses,
+        "constraints": {BRICK.hasPart: [BRICK.Equipment, BRICK.Point]},
+    },
+    "Photovoltaic_Array": {
```

Comment:
Good catch! Should be fixed for the PV stuff

The question for whether an AHU is a system is one of opinion between the standards. For us, it is not a system but it would be a member of the HVAC system.